python/identify_filenames.py

```diff
@@ -11,8 +11,8 @@
 examined for every feature.
 """
 
-import platform
-if platform.python_version_tuple() < ('3','2','0'):
+import sys
+if sys.version_info < (3,2):
```

Comment:
At this point, this can be removed. Nobody is using Python 2.x or versions less than Python 3.2. Let's just delete the code?

Reply:
It might be worthwhile to keep it in for whatever reason someone might be using < 3.2 but yes, it's unlikely they would be.